apps/src/templates/projects/PersonalProjectsTable.jsx

```diff
@@ -8,7 +8,7 @@ import wrappedSortable from '../tables/wrapped_sortable';
 import orderBy from 'lodash/orderBy';
 import {PROJECT_TYPE_MAP, personalProjectDataPropType} from './projectConstants';
 import QuickActionsCell from '../tables/QuickActionsCell';
-import ProjectActionBox from './ProjectActionBox';
+import QuickAction from '../tables/QuickAction';
```

Comment:
after removing ProjectActionBox from this file, it looks like it is only used by ProjectCard.jsx (and it's own storybook entry). This seems fine, just confirming my understanding.

Reply:
Yes, that's correct. Mostly just didn't want to remove any storybook functionality from the project card, and this is at least as good as it was before.